### PR TITLE
Add environment variables on container specs

### PR DESCRIPTION
**Pull request by Ripolin**
#### Why is this pull request needed and what does it do?

Finish work started in #90 (rebase + bump version)

### CODE REVIEW

File: charts/coredns/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: coredns
-version: 1.35.1
+version: 1.35.2
```
Comment (hagaibarel) [inline charts/coredns/Chart.yaml:3]:
```suggestion
version: 1.36.0
```

Also, please update the changes under the annotations section

**Review by hagaibarel — CHANGES_REQUESTED** [review]

**Comment by hagaibarel** [comment]
@Ripolin the DCO check is failing, please fix the problematic commits. 

Also, please update the changes section in chart.yaml

**Comment by Ripolin** [comment]
> @Ripolin the DCO check is failing, please fix the problematic commits.
> 
> Also, please update the changes section in chart.yaml

Should be OK now

**Review by hagaibarel — CHANGES_REQUESTED** [review]
Please modify the changes under annotations (L21)

**Comment by Ripolin** [comment]
> Please modify the changes under annotations (L21)

Done

**Review by hagaibarel — APPROVED** [review]

**Comment by hagaibarel** [comment]
Thanks for the PR!

**Pull request merged**